Tools/TestWebKitAPI/Tests/WebKitCocoa/TLSDeprecation.mm

```diff
@@ -466,19 +461,17 @@ HTTPServer mixedContentServer({
 
     [webView loadRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:[NSString stringWithFormat:@"https://localhost:%d/", mixedContentServer.port()]]]];
     [delegate waitForDidFinishNavigation];
-    while ([webView hasOnlySecureContent])
-        TestWebKitAPI::Util::spinRunLoop();
-    EXPECT_FALSE([webView hasOnlySecureContent]);
+    TestWebKitAPI::Util::spinRunLoop(20);
```

Comment:
We could delete this test, or adapt it in another way to verify that we didn't load any insecure content. I don't think this test will be meaningful in its new form.